src/capi/azext_capi/_help.py

```diff
@@ -74,6 +74,13 @@
 short-summary: Delete a workload cluster.
 long-summary: |
     See https://capz.sigs.k8s.io/ for more information.
+parameters:
```

Comment:
This parameters section isn’t necessary. The `—name` and `—resource-group` args already have default descriptions set elsewhere that you can see in `az capi delete —help`.